src/cascadia/TerminalCore/TerminalDispatchGraphics.cpp

```diff
@@ -69,30 +69,28 @@ bool TerminalDispatch::s_IsDefaultColorOption(const DispatchTypes::GraphicsOptio
 //      RGB sequences then take 3 MORE params to designate the R, G, B parts of the color
 //      Xterm index will use the param that follows to use a color from the preset 256 color xterm color table.
 // Arguments:
-// - rgOptions - An array of options that will be used to generate the RGB color
-// - cOptions - The count of options
-// - pcOptionsConsumed - a pointer to place the number of options we consumed parsing this option.
+// - options - An array of options that will be used to generate the RGB color
+// - optionsConsumed - Returns the number of options we consumed parsing this option.
 // Return Value:
 // Returns true if we successfully parsed an extended color option from the options array.
 // - This corresponds to the following number of options consumed (pcOptionsConsumed):
 //     1 - false, not enough options to parse.
 //     2 - false, not enough options to parse.
 //     3 - true, parsed an xterm index to a color
 //     5 - true, parsed an RGB color.
-bool TerminalDispatch::_SetRgbColorsHelper(_In_reads_(cOptions) const DispatchTypes::GraphicsOptions* const rgOptions,
-                                           const size_t cOptions,
-                                           _Out_ size_t* const pcOptionsConsumed)
+bool TerminalDispatch::_SetRgbColorsHelper(const std::basic_string_view<DispatchTypes::GraphicsOptions> options,
+                                           size_t optionsConsumed)

```

Comment:
Isn't `optionsConsumed` meant to be a reference now? Otherwise it's not going to be returned.

Reply:
YEP. Great catch. Thank you. It's a bit concerning a test didn't catch this... might need to add one.

Reply:
Looks like there's no real testing library on this already. I think #3849 should probably take on that work.